src/main/java/ratpack/zipkin/internal/RatpackCurrentTraceContext.java

```diff
@@ -27,19 +24,54 @@
 import ratpack.registry.MutableRegistry;
 
 public final class RatpackCurrentTraceContext extends CurrentTraceContext {
+  public static CurrentTraceContext create(){
+    return new Builder().build();
+  }
+
+  public static Builder newBuilder(){
+    return new Builder();
+  }
+
+  public static final class Builder extends CurrentTraceContext.Builder {
+    Supplier<MutableRegistry> registrySupplier = Execution::current;
+
+    public Builder registrySupplier(Supplier<MutableRegistry> registrySupplier) {
```

Comment:
@hyleung @llinder curious.. if this is only for tests.. maybe we don't need to expose this public? Is this used elsewhere?